spec/features/hq/navigation_menu.spec.rb

```diff
@@ -0,0 +1,57 @@
+require 'rails_helper'
+require 'spec_helper'
+
+RSpec.feature 'Navigation menu', :type => :feature do
+    
+  background do
+    i_sign_in_as_admin
+  end
+  scenario 'user cilco on OSRA link' do
+    within "#main_nav_bar" do
+      click_link "OSRA"
+    end
+   # expect(page).to eq hq_index_path
+  end
```

Comment:
better add a `pending` here instead of only a comment so we know to come back when we have the correct path.
when you run `rspec` the pending tests are displayed at the end in orange/yellow